src/demo/demos.py

```diff
@@ -240,36 +250,45 @@ def predict_MerchantSize_on_lead_data_demo():
     import pandas as pd
 
     log.info(
-        "Note: Enriched data must be located at s3://amos--data--events/leads/enriched.csv"
+        "Note: In case of running locally, enriched data must be located at src/data/leads_enriched.csv locally\nIn case of running on S3, enriched data must be located at s3://amos--data--events/leads/enriched.csv or"
```

Comment:
I think you could remove the second "locally" and should remove the "or" at the end

Reply:
Addressed.